tests/unit/updateTestResults.unitTest.ts

```diff
@@ -0,0 +1,228 @@
+import {TestResultsService} from "../../src/services/TestResultsService";
+import {HTTPError} from "../../src/models/HTTPError";
+import fs from "fs";
+import path from "path";
+import {MESSAGES} from "../../src/assets/Enums";
+import {cloneDeep} from "lodash";
+
+describe("updateTestResults", () => {
+    let testResultsService: TestResultsService | any;
+    let MockTestResultsDAO: jest.Mock;
+    let testResultsMockDB: any;
+    let testToUpdate: any;
+    const msUserDetails = {
+        msUser: "dorel",
+        msOid: "123456"
+    };
+    beforeEach(() => {
+        testResultsMockDB = JSON.parse(fs.readFileSync(path.resolve(__dirname, "../resources/test-results.json"), "utf8"));
```

Comment:
suggest to `import testResults from "../resources/test-results.json"` then map to `testResultsMockDB`

Reply:
done